### DISCUSSION TITLE
Standalone clickhouse-local

### DISCUSSION THREAD

**Discussion by akizminet** [ClickHouse/ClickHouse#39914] [Q&A] [answered]
Is there any way to download & run clickhouse-local without installing clickhouse?

**Comment by DerekChia** [comment, marked as answer]
You can download the single binary by following the instruction here https://clickhouse.com/docs/en/quick-start/

After which you can run clickhouse-local like this:

```
> ./clickhouse local
ClickHouse local version 22.8.1.962 (official build).

dereks-macbook-pro.local :)
```
